src/In.ProjectEKA.OtpService/Otp/OtpGenerationRequest.cs

```diff
@@ -3,13 +3,14 @@ namespace In.ProjectEKA.OtpService.Otp
     public class OtpGenerationRequest
     {
         public string SessionId { get; set; }
-
         public Communication Communication { get; set; }
+        public OtpCreationDetail CreationDetail { get; set; }
```

Comment:
Class name is OtpGenerationRequest and property name is OtpCreationDetail.

can we change this to OtpGenerationDetail, because we won’t create OTP, we generate it

Reply:
done